### DISCUSSION TITLE
Assistance Needed: Jobrunr Implementation Issue with CockroachDB and Spring Boot

### DISCUSSION THREAD

**Discussion by Anvarbekovich** [jobrunr/jobrunr#947] [Q&A]
**Hi Jobrunr Experts,

I hope this message finds you well. We're currently encountering some challenges with integrating Jobrunr with CockroachDB in our Spring Boot 3+ application using JDK 17.

We are using version 6.3.3 of the Jobrunr Spring Boot 3 starter:**

```
<dependency>
    <groupId>org.jobrunr</groupId>
    <artifactId>jobrunr-spring-boot-3-starter</artifactId>
    <version>6.3.3</version>
</dependency>
```
**When attempting to run with default property values, we're encountering the following error:**
`org.postgresql.util.PSQLException: ERROR: at or near "(": syntax error
`

Additionally, when skipping database creation and using SQL scripts from the source code manually, we're facing the following exception:
```
java.lang.UnsupportedOperationException: Cannot autobox 1 of type java.lang.Long to java.lang.Boolean
    at org.jobrunr.utils.reflection.autobox.BooleanTypeAutoboxer.autobox(BooleanTypeAutoboxer.java:22)
    at org.jobrunr.utils.reflection.autobox.BooleanTypeAutoboxer.autobox(BooleanTypeAutoboxer.java:7)
    at org.jobrunr.utils.reflection.autobox.Autoboxer.lambda$autobox$1(Autoboxer.java:37)
    at java.base/java.util.Optional.map(Optional.java:260)
```
**We've tried various troubleshooting steps, but haven't been successful in resolving these issues. We'd greatly appreciate any insights, recommendations, or guidance you could provide to help us overcome these challenges.

If possible, could you please advise on the best practices for integrating Jobrunr with CockroachDB in a Spring Boot 3+ application using JDK 17?

Thank you very much for your assistance. We're looking forward to your response.**

**Comment by rdehuyss** [comment]
👋 @Anvarbekovich,

I'm afraid we currently don't support CockroachDB (as you might have guessed).

If your company would become a customer of JobRunr Pro, we may consider adding it. I hope you understand that setting up a Cockroach DB and testing JobRunr OSS against it takes a lot of our time...
